### DISCUSSION TITLE
The mobile screenshots are too large, and this distorts the HTML repot

### DISCUSSION THREAD

**Discussion by zbrea** [getgauge/gauge#2237] [Ideas]
** The problem is that, in the event of a bug reported in the html reporting, the screenshot of the bug becomes very very large, because it fits with the page which becomes very large because the data table has a lot of values.

So to avoid this problem I suggest you add a scroll, if the width of the data table becomes wider than the html reporting web page.

I propose to add a scroll to the html reporting for the shared data table in the scenario of the same specification, so that it adapts to a standard display**
I use a lot of data in an external csv file, for each specification (covered by several test scenarios), for a mobile application.

**1- Problem:**
![report_Problem](https://user-images.githubusercontent.com/59898499/164913202-4db5639f-0dff-4e10-88c4-8fcc08450f09.PNG)

**2- Solution:** see the scroll pointed by the red arrow under the table
![report_Solution](https://user-images.githubusercontent.com/59898499/164913734-af167b2a-ebec-4bfd-a42b-faaa99add87d.PNG)

**Comment by zbrea** [comment]
Can you tell me in which method or class, I can fix this bug in the source code locally at home?
As a reminder, the bug is that the mobile screenshots are too large, and this distorts the HTML repoting

**Reply by zabil** [reply]
This is handled by https://github.com/getgauge/html-report you can take a look there.
Here is the part the handles the screenshot
https://github.com/getgauge/html-report/blob/7b9582e1532a5efb4cca95e254b385c42045b973/themes/default/views/partials.tmpl#L155
